src/jquery.tabletojson.js

```diff
@@ -102,9 +102,19 @@
 
           $row = $(row);
 
-          var isEmpty = ($row.find('td').length === $row.find('td:empty').length) ? true : false;
+          var isEmptyRow = function (row){
+            let result = true;
+            let cells = $(row).find('td').not(ignoredColumn);
+            for (let i = 0; i < cells.length; i++) {
+              if (cellValues(i, cells[i], false) != ''){
+				  result = false;
+				  break;
+			  }
+            }
+            return result;
+          }
```

Comment:
Missing `;`

Reply:
Also don't use tabs
![2020-01-17 07_35_34-BUGFIX_ ignoreEmptyRows by cn-tools · Pull Request #54 · lightswitch05_table-to-](https://user-images.githubusercontent.com/136959/72616220-0c730500-38fc-11ea-86f9-e6b5d77b4773.png)